lib/Onyx.js

```diff
@@ -1,7 +1,8 @@
 import _ from 'underscore';
-import AsyncStorage from '@react-native-async-storage/async-storage';
 import Str from 'expensify-common/lib/str';
 import lodashMerge from 'lodash/merge';
+import Storage from './storage';
```

Comment:
Instead of AsyncStorage or MMKV we use the `Storage` facade

Reply:
Would love to see a simpler version of this PR where we do something like

`import AsyncStorage from './AsyncStorage`

Then have 

`index.js` - import / export from `@react-native-async-storage/async-storage`
`index.native.js` - export a custom `AsyncStorage`

Reply:
I don't think we should use the name AsyncStorage if the underlying storage implementation could be of a different library. A more generic term should be used so it's clear we're not tied to AsyncStorage

Reply:
Not sure it matters for the purposes of the POC - the code will run the same there will just be less of it.